Lesson 6/Lesson 6/AccountTypes.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Lesson6
+{
+    enum AccountTypes
```

Comment:
```suggestion
    internal enum AccountTypes
```
Модификатор доступа потерялся.